UPGRADE-GUIDE.md

```diff
@@ -0,0 +1,1298 @@
+- `--pf-c-alert--m-inline--m-warning__icon--Color`
+- `--pf-c-alert--m-inline--m-warning--before--BackgroundColor`
+- `--pf-c-alert--m-inline--m-info__icon--Color`
+- `--pf-c-alert--m-inline--m-info--before--BackgroundColor`
+
+### Application launcher
+- Applied external link icon class to icon wrapper [(#2904)](https://github.com/patternfly/patternfly/pull/2904)
+- Removed separator in favor of divider component [(#2944)](https://github.com/patternfly/patternfly/pull/2944)
+- Removed hover, active, focus, and disabled from workspace [(#2975)](https://github.com/patternfly/patternfly/pull/2975)
+- Cleaned up variables [(#3012)](https://github.com/patternfly/patternfly/pull/3012)
+
+Updated markup:
+- Moves the class `.pf-c-app-launcher__menu-item-external-icon` from the external link icon itself to a `<span>` that wraps the external link icon. Any instances of the external link icon should be wrapped in a `<span>` and the `.pf-c-app-launcher__menu-item-external-icon` class should be moved from the icon to the `<span>`.
+
+Removed classes:
+- `.pf-c-app-launcher__separator` - element was removed. Use the divider component instead.
+- `.pf-m-hover` from `.pf-c-app-launcher__toggle` and `.pf-c-app-launcher__menu`
+- `.pf-m-focus` from `.pf-c-app-launcher__toggle` and `.pf-c-app-launcher__menu`
+- `.pf-m-disabled` from `.pf-c-app-launcher__toggle`
+  - The `:hover`, `:active`, `:focus`, and `:disabled` selectors still have styles applied to them
+
+Renamed variables:
+- `--pf-c-app-launcher--m-top__menu--Transform` to `--pf-c-app-launcher--m-top__menu--TranslateY`
+- `--pf-c-app-launcher__menu-item-external-icon--Transform` to `--pf-c-app-launcher__menu-item-external-icon--TranslateY`
+
+Removed variables:
+- `--pf-c-app-launcher__group--PaddingTop`
+- `--pf-c-app-launcher__group--first-child--PaddingTop`
+- `--pf-c-app-launcher__menu--BorderWidth`
+- `--pf-c-app-launcher__separator--Height`
+- `--pf-c-app-launcher__separator--BackgroundColor`
+- `--pf-c-app-launcher__separator--MarginTop`
+- `--pf-c-app-launcher__separator--MarginBottom`
+- `--pf-c-app-launcher__separator--Height`
+
+### Backdrop
+- Removed blur [(#3009)](https://github.com/patternfly/patternfly/pull/3009)
+
+Renamed variables:
+- `--pf-c-backdrop--Color` to `--pf-c-backdrop--BackgroundColor`
+
+Removed variables:
+- `--pf-c-backdrop--BackdropFilter`
+
+### Breadcrumb
+- Added word-break, display link and icon inline, move divider [(#2916)](https://github.com/patternfly/patternfly/pull/2916)
+- Removed hover, active, focus, and disabled from workspace [(#2975)](https://github.com/patternfly/patternfly/pull/2975)
+- Made redhat font default, add opt-in for overpass [(#2955)](https://github.com/patternfly/patternfly/pull/2955)
+
+Updated markup:
+- Any reference to the divider element `.pf-c-breadcrumb__item-divider` will need to be moved in the markup to come before the breadcrumb link element `.pf-c-breadcrumb__link` instead of after it.
```

Comment:
```suggestion
- In the markup, move any reference to the divider element `.pf-c-breadcrumb__item-divider` so that is comes before the breadcrumb link element `.pf-c-breadcrumb__link` instead of after it.
```